src/lib/link.ts

```diff
@@ -37,6 +37,11 @@ export function otherSide(side: Side): Side {
     return 'A';
   }
 }
+/**
+ * PacketFilter is the type for a function that accepts a Packet and returns a boolean definign whether to relay the packet or not
+ */
+
+export type PacketFilter = (packet: Packet) => boolean;
```

Comment:
This empty line between doc comment and type seems off. It probably will cause the doc comment to not be found